src/NServiceBus.Testing.Fakes/obsoletes-v8.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus.Testing
+{
+    using System;
+
+    public partial class TestableTransportReceiveContext// : TestableBehaviorContext, ITransportReceiveContext
+    {
+        [Obsolete("The AbortReceiveOperation method has been removed. See the upgrade guide for more details. Will be removed in version 9.0.0.", true)]
+        public bool ReceiveOperationAborted { get; set; }
+
+        [Obsolete("The AbortReceiveOperation method has been removed. See the upgrade guide for more details. Will be removed in version 9.0.0.", true)]
+        public virtual void AbortReceiveOperation()
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
very minor, but this can be

```suggestion
        public virtual void AbortReceiveOperation() => throw new NotImplementedException();
```